projects/ion/src/lib/notification/notification.component.ts

```diff
@@ -53,7 +58,10 @@ export class NotificationComponent {
   }
 
   closeNotification() {
-    this.notification.nativeElement.remove();
+    this.notification.nativeElement.classList.add(this.fadeOut);
+    setTimeout(() => {
+      this.notification.nativeElement.remove();
+    }, 1000);
```

Comment:
Why this timeout?

Reply:
This way the element will only be removed when the animation is complete. Without this TimeOut the "remove" action just destroys the component before the animation, interrupting the animation and sudenly disapearing with the element in the View

Reply:
setTimeout can be replaced by delay operator of RxJS